File: D_Content/Day10/D10_4_ActivityPractice15/app/src/main/java/org/techtown/ap15/MainActivity.java
package org.techtown.ap15;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.LinearLayout.LayoutParams;

public class MainActivity extends AppCompatActivity {

    ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        imageView = findViewById(R.id.imageView);

        LayoutParams params = (LayoutParams) imageView.getLayoutParams();
        params.height = params.width;
        imageView.setLayoutParams(params);


    }
}

File: D_About_Kotlin/03_AndroidByKotlin/03_HelloAndroid/app/src/main/java/org/techtown/helloandroid/MainActivity.kt
package org.techtown.helloandroid

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //val tv_Title = findViewById<TextView>(R.id.tv_Title)
        tv_Title.text = "Changed Text"
    }
}